src/deploy-cloudrun.ts

```diff
@@ -131,7 +131,9 @@ export async function run(): Promise<void> {
       // Set optional flags from inputs
       if (envVars) cmd.push('--update-env-vars', envVars);
       if (secrets) {
-        cmd.push('--update-secrets', secrets);
+        // If there are spaces not within quotes 
+        const regex = /[ ]+(?=[^"]*(?:"[^"]*"[^"]*)*$)/;
```

Comment:
Would it be simpler to split on newline and construct the value?

Reply:
From my tests the input string would be: `INPUT_1=secret1       INPUT_2=secret1` instead of the expected `INPUT_1=secret1\nINPUT_2=secret1`.